src/UI/Art/SpriteArtist/CommanderOverlayArtist.java

```diff
@@ -67,62 +73,69 @@ public static int drawCommanderOverlay(Graphics g, Army army, int yInitialOffset
     final int coEyesWidth = 25;
     final int xTextOffset = (4+coEyesWidth); // Distance from the side of the view to the CO overlay text.
     final int yTextOffset = 3; // Distance from the top of the view to the CO overlay text.
+    final int yTextOffsetLineTwo = SpriteLibrary.baseIconSize + yTextOffset - 1; // Ditto, but for line 2
     final int mapViewWidth = SpriteOptions.getScreenDimensions().width / SpriteOptions.getDrawScale();
     final BufferedImage spriteA = SpriteLibrary.getLettersSmallCaps().getFrame(0); // Convenient reference so we can check dimensions.
 
-    int yCurrentOffset = yInitialOffset;
+    final int yShiftPerOverlay = SpriteLibrary.getCoOverlay(army.cos[0], overlayIsLeft).getHeight() - 1;
+
+    final int fundsLength = ("" + army.money).length();
+    final int fundsWidth = ICON_VALUE_SPACING + fundsLength * spriteA.getWidth() + 1;
+    final int drawX = overlayIsLeft ? 0 : mapViewWidth - fundsWidth;
+    final int drawY = yInitialOffset + yShiftPerOverlay*army.cos.length + yTextOffset;
+    SpriteUIUtils.drawMenuFrame(g, SpriteUIUtils.MENUBGCOLOR, SpriteUIUtils.MENUFRAMECOLOR,
+        drawX, drawY, fundsWidth, fundsHeight, moneyBufferPx);
+    drawIconAndValue(g, SpriteLibrary.MapIcons.FUNDS, army.money, fundsLength, drawX, drawY + moneyBufferPx);
```

Comment:
A couple comments:
1: I like putting the power bar back underneath the CO portrait, but in team merge mode you still need to add a bit of buffer between the two overlays so the bar doesn't cover the next CO's forehead.
2: Tossing the funds count in a differently-formatted frame underneath the CO portrait looks odd. Since we already have the faces to say who's who, can we put funds back inside the overlay area instead of the name? Yes, this would mean all COs would report the same amount in merge mode, but I don't think that's really a bad thing. If anything, it just underscores that they have a shared purse.

Reply:
Fair 'nuff.
I don't like putting redundant info on the GUI, but I guess I'll try it.